docs/db/migrations.md

```diff
@@ -154,6 +154,44 @@ type Note {
 
 This maps to `comment.note_id` in the database.
 
+##### ManyToMany
+
+To create a many-to-many relationship, add a model for your join table and use two one-to-many relationship mappings to create the relationship.
+
+```graphql
+""" 
+@model 
+"""
+type Note {
+  id: ID!
+  title: String!
+  description: String
+  """
+  @oneToMany field: 'note'
+  """
+  authors: [NoteAuthor]
+}
+
+"""
+@model
+"""
+type NoteAuthor {
```

Comment:
So generally people will get a lots of pointless boilerplate generated for this - entire crud and will need to manually remove all CRUD capabilities for each type like that.